### ISSUE TITLE
Chat mentions should display the user's display name instead of id

### ISSUE THREAD

**Issue by younesaassila** [benawad/dogehouse#368]

**Comment by benawad** [comment]
I want it to display the username

**Comment by younesaassila** [comment]
The problem is that some people have IDs very different from their Display Name, making it very hard to follow chat.

**Comment by nadirabbas** [comment]
@younesaassila Twitch does the same. They show usernames.

**Comment by younesaassila** [comment]
> @younesaassila Twitch does the same. They show usernames.

Twitch is different though because they don't have display names. They only allow you to customize the capitalization of your username.

You could argue that Facebook for example does show the display name and not the username when mentioning someone.

**Comment by nadirabbas** [comment]
We can show usernames instead of display names in chat too. As @benawad wants to clone the Twitch chat.

**Comment by younesaassila** [comment]
> We can show usernames instead of display names in chat too. As @benawad wants to clone the Twitch chat.

That would be a problem because then the name shown below your avatar in the Listeners/Speakers section wouldn't match your name in chat.
We either have to show display names in mentions or get rid of them entirely and only display usernames.
